python/fastquant/strategies/base.py

```diff
@@ -289,6 +294,32 @@ def next(self):
                         self.sell(exectype=bt.Order.StopTrail, trailpercent=self.stop_trail, size=final_size)
 
         elif self.sell_signal():
+            if self.allow_short == True:
+
+                # Sell short based on the closing price of the previous day
+                if self.execution_type == "close":
+
+                    if self.transaction_logging:
+                        self.log("SELL CREATE, %.2f" % self.dataclose[0])
+
+                    sell_prop_size = int(SELL_PROP * 
+                                         self.broker.getvalue() / 
+                                         self.dataclose[0])
+                    self.order = self.sell(size=sell_prop_size)
+                
+                 # Buy based on the opening price of the next closing day (only works "open" data exists in the dataset)
+                else:
+
+                    if self.transaction_logging:
+                        self.log("SELL CREATE, %.2f" % self.dataclose[1])
+                    
+                    sell_prop_size = int(self.short_max *
+                                         self.cash /
+                                         self.dataclose[1] /
+                                         (1 + self.commission + 0.001))
+                    self.order = self.sell(size=sell_prop_size)
```

Comment:
```suggestion
                    self.order = self.sell(size=min(sell_prop_size, self.init_cash * SHORT_MAX))
```

Reply:
adding this minimum line so that the maximum short position is equal to `self.init_cash * SHORT_MAX`.

I.e. if `SHORT_MAX` = 1.5, then the maximum short position is the initial cash position (e.g. 100000) times 1.5, which is 150000.